cvmfs/receiver/payload_processor.cc

```diff
@@ -0,0 +1,118 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "payload_processor.h"
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <vector>
+
+#include "../logging.h"
```

Comment:
This can be simply `#include "logging.h"`, the cvmfs directory is in the default search paths.